src/adr/0030-separate-attachment-content-and-metadata.md

```diff
@@ -0,0 +1,84 @@
+# 30. Separate attachment content and metadata
+
+Date: 2020-04-13
+
+## Status
+
+Accepted (lazy consensus)
+
+## Context
+
+Some mailbox implementations of James store already parsed attachments for faster retrieval.
+
+Here are the POJOs representing these attachments:
```

Comment:
the list doesn't contain POJOs representing these attachments, it's more the list of POJOs related to attachements